CMakeLists.txt

```diff
@@ -158,6 +158,31 @@ if(APPLE AND ZIG_STATIC)
     list(APPEND LLVM_LIBRARIES "${CURSES}")
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" AND ZIG_STATIC)
```

Comment:
This FreeBSD stuff belongs into another PR or the PR should explain the fix properly with problem, fix and outcome.